frontend/src/theme/components.scss

```diff
@@ -39,3 +37,11 @@ h2 {
 h4 {
   text-align: center;
 }
+
+.el-table th.el-table__cell {
+  color: $color-black;
+}
+
+.el-button--primary {
+  box-shadow: 2px 3px 7px 0px #2e2e2e33;
```

Comment:
Remember: you should use variables for colors. Use one of greys that already exists